felix/fv/infrastructure/topology.go

```diff
@@ -38,8 +38,11 @@ import (
 )
 
 type TopologyOptions struct {
-	FelixLogSeverity          string
-	EnableIPv6                bool
+	FelixLogSeverity string
+	EnableIPv6       bool
+	// Temporary flag to implement and test IPv6 in bpf dataplane.
+	// TODO: Remove it when IPv6 implementation in BPF mode is complete.
```

Comment:
Who knows how temporary it is going to be? :man_shrugging: :-)

Reply:
I think, when the feature is in tech preview and announced, we can switch back to the original config. WDYT?